packages/nodejs/src/computers/JsonFileWrite/JsonFileWrite.ts

```diff
@@ -26,9 +26,14 @@ export const JsonFileWrite: Computer = {
   async *run({ input, params }) {
     const incoming = input.pull()
 
-    const path = params.path as string
+    const filePath = params.file_path as string
     const content = JSON.stringify(incoming.map(i => i.value), null, 2)
 
-    await fs.writeFile(path, content, 'utf-8')
+    const root = process.env.WORKSPACE_FOLDER_PATH;
+    if(!root) throw new Error('WORKSPACE_FOLDER_PATH not set')
```

Comment:
Have you considered putting the computers related to the Workspace in a single directory? This is only relevant in the Workspace mode of the VSCode extension.